tfs/writer.py

```diff
@@ -30,11 +31,12 @@ def write_tfs(
     colwidth: int = DEFAULT_COLUMN_WIDTH,
     headerswidth: int = DEFAULT_COLUMN_WIDTH,
     non_unique_behavior: str = "warn",
-    validate: bool = True,  # noqa: FBT001, FBT002
+    validate: str = "madx",
```

Comment:
**Note:** Validation is enabled in `MAD-X` compatility mode by default before writing (since it is the most restrictive).